wordpress-mobile/WordPress-Android
# Fix squashed format bar icons in visual editor

Fixes #6150, restoring `44dp` icons for use by the visual editor only, and using a separate `dimen` for the visual editor's format bar height.

Also fixes a visual issue with the new photo picker in the legacy editor, where extra space was shown above the format bar with the picker active (this was due to the legacy format bar being `40dp`, while the picker used `44dp` for its format bar overlay, which is what the visual and Aztec editors used to both use):

![legacy-photo-picker-space](https://user-images.githubusercontent.com/9613966/27559115-57cc0808-5a8d-11e7-8d60-715e68207ed4.png)

To test:
Verify that the format bar looks right for:
* The legacy editor
* The visual editor at the different sizing buckets: `< 360dp`, `360dp - 379dp`, `380dp - 599dp`, `> 600dp` (AS's design view for XML files helps with this as well)
* Aztec
Also verify that things look normal when the photo picker is shown in all three editors.

cc @theck13, and @nbradbury as this affects the photo picker
